### DISCUSSION TITLE
Resizing window does an odd painting

### DISCUSSION THREAD

**Discussion by leonardo-pascual** [ohmyzsh/ohmyzsh#10511] [Help] [answered]
![Screen Shot 2021-12-15 at 10 53 42 AM](https://user-images.githubusercontent.com/91507056/146230246-53187c13-fbf5-42b1-b153-d82bfeb933d7.png)

**Comment by mcornella** [comment, marked as answer]
This is expected behavior. The prompt can't delete it's lines when it resizes. You can press CTRL-L to clear the terminal window and get a fresh new prompt line.

**Reply by huangcs427** [reply]
This link might help you.
https://github.com/romkatv/powerlevel10k#horrific-mess-when-resizing-terminal-window
